### DISCUSSION TITLE
Disable displaying date

### DISCUSSION THREAD

**Discussion by chipoman** [ente-io/ente#3859] [Q&A]
Does anyone know, is it possible to  disable displaying the date above photos, or change it to title/description? I have many old photo that I have scanned from albums. The date displayed is when I scanned the photo, not when it was taken and is thus misleading.

**Comment by mnvr** [comment]
On the web and desktop app, there is an option to "Fix dates" (select one or more photos, and in the selection bar you'll see option, it has the clock icon). After opening this, it will provide options to select alternative Exif fields where it will pick the date from. Maybe one of these alternative fields might be the appropriate one to use here.

**Reply by chipoman** [reply]
Thank you for the reply. My photo collection starts in the 1950's, long before anyone had thought of EXIF data! The EXIF data relates to the date the images were scanned, long after the early photos were taken. I have used the file name to add dates to all the files.  All the filenames have the format 'yyyy-mm-dd n description'. This enables me to have all the files in folders organised by date. The 'n' in the filename enables me to organise photos within days. So, all my photos can immediately be ordered correctly if sorted by filename.  I have loaded some photos from the 1960's into Ente as an experiment. They are all out of order, being arranged by the date in which they were scanned. They are displaying dates ranging from 2014 onward, which is very confusing when the photo was taken in the 1960's. There are hundreds of old photos, so it is not a realistic job to redate them all.

**Reply by mnvr** [reply]
That option supports bulk fixing. If you try out the option that I mentioned, you'll see that it will provide you ways of picking an alternative Exif field to use as the photo's data. 
<img width="332" alt="Screenshot 2024-10-30 at 08 05 18" src="https://github.com/user-attachments/assets/2bac4669-64c8-497a-9a15-37ffab7f1b86">

So suppose the appropriate field for your photos is "Exif:DataTimeDigitized" (so you try all the options on a sample photo first to find the correct one), then you can select all the photos you want to update, select the appropriate date option, and it will fix them all in the same go.

If none of these fields are the correct one in your photo, then you could extend the script we had discussed https://github.com/ente-io/ente/discussions/3857#discussioncomment-11094505 for extracting the description, and also extract the appropriate Exif date field for your dataset and put it in the JSON file. If the JSON file has a date, that will be preferred over the Exif ones when importing the photo, so this way you can use any date as wished.

Here is a sample of how the JSON should look:
```
{
  "description": "Test",
  "creationTime": {
    "timestamp": "1613532136",
    "formatted": "17 Feb 2021, 03:22:16 UTC"
  }
}
```

**Reply by mnvr** [reply]
> My photo collection starts in the 1950's

❤️
